packages/logger/tests/unit/Logger.test.ts

```diff
@@ -1419,7 +1419,7 @@ describe('Class: Logger', () => {
       };
       const childLoggerWithSampleRateEnabled = parentLogger.createChild(optionsWithSampleRateEnabled);
 
-      const optionsWithErrorLogLevel = {
+      const optionsWithErrorLogLevel: ConstructorOptions = {
```

Comment:
Just a small nitpick, the name for the Logger constructor is too generic, so it's probably better to align it with other modules to `LoggerOptions`.

Reply:
Agreed, we should revisit all the Logger types at some point in the future, for instance [this one](https://github.com/awslabs/aws-lambda-powertools-typescript/blob/main/packages/logger/src/types/Logger.ts#L7).

However I think it'd be a breaking change, so it's going to be a v2 discussion.

Reply:
[Added a comment](https://github.com/awslabs/aws-lambda-powertools-typescript/discussions/1269#discussioncomment-5114490) to the v2.0.0 milestone discussion.